### PR TITLE
Ability to disable manage_service if installed from package

### PR DESCRIPTION
**Pull request by Samgarr**

### CODE REVIEW

**Comment by bastelfreak** [comment]
@Samgarr thanks for the PR. @tuxmea what do you think about this solution? I would prefer it over https://github.com/voxpupuli/puppet-prometheus/pull/100

**Comment by tuxmea** [comment]
I would recommend spec tests also for the new parameter. Besides this: looks nicer than mine.

File: manifests/daemon.pp [hunk 1]
```diff
@@ -75,7 +75,7 @@
   $init_style         = $::prometheus::params::init_style,
   $service_ensure     = 'running',
   $service_enable     = true,
-  $manage_service     = true,
+  $manage_service     = $::prometheus::params::manage_service,
```
Comment (tuxmea) [inline manifests/daemon.pp]: The parameter should be taken from $::prometheus::manage_service ! Not taking default from params.pp

**Review by tuxmea — CHANGES_REQUESTED** [review]

File: manifests/config.pp [hunk 2]
```diff
@@ -8,7 +8,7 @@
   $config_template = $::prometheus::params::config_template,
 ) {
 
-  if $prometheus::init_style {
+  if $prometheus::init_style and $prometheus::manage_service == true {
 
     # the vast majority of files here are init-files
```
Comment (tuxmea) [inline manifests/config.pp]: This ssems to be an outdated version of config.pp. please rebase!

**Review by tuxmea — CHANGES_REQUESTED** [review]

**Comment by bastelfreak** [comment]
Hi @Samgarr, could you please rebase this again?

**Comment by Samgarr** [comment]
Hi, i could rebase, but isn't this issue already solved?

**Comment by bastelfreak** [comment]
Ah true, we now have a `manage_service` parameter

**Pull request closed without merging**